### DISCUSSION TITLE
Can't choose OpenGAPPS or MineTheGAPPS ?

### DISCUSSION THREAD

**Discussion by ghost** [LSPosed/MagiskOnWSALocal#194] [Q&A] [answered]
I want to choose OpenGAPPS but why secret MineTheGAPPS ?

**Comment by Howard20181** [comment, marked as answer]
Use the up and down arrow keys to move
Spacebar to select

**Reply by ghost** [reply]
I use the keyboard 's arrow to choose opition but nothing happens
